IsogeomGenerator/vol.py

```diff
@@ -979,11 +841,45 @@ def _tag_for_viz(self):
 
```

Comment:
PEP 257 recommends no blank lines before or after a docstring
```suggestion
```

Reply:
I think this was just displaying weirdly. There is not a blank line there.